### PR TITLE
Making normalize call take an uploaded file instead of XML in the req…

### PR DESCRIPTION
**Pull request by tingulfsen**
…uest itself.

### CODE REVIEW

**Comment by tingulfsen** [comment]
This pull request brings the normalize request processing into line with the modsulator request, by accepting an uploaded file of XML instead of taking its input from the request body. The main reason for this is just that this enables us to use the same code on the client side for the two requests.

I've also removed ruby-head from the Travis build configuration. The unf_ext gem won't build under ruby-head. I don't think it's particularly important for us to work perfectly with the very latest Ruby version, so I just removed it instead of spending time on debugging a fairly minor issue that doesn't affect functionality (we run 2.2.4 on the server anyway).

Hopefully this Grape application will soon be superseded by a Rails API, which is much easier to work with (and understand).

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 language: ruby
 script: bundle exec rake
 rvm:
-  - ruby-head
```
Comment (ndushay) [inline .travis.yml:4]: sometimes I comment it out and explain why it's not there.  This allows folks to know if it is safe to add it back (like if a bug is fixed):

```
#   - ruby-head # unf_ext gem won't build under ruby-head
```

probably ok not to worry about it here.

**Comment by ndushay** [comment]
Maybe make a note somewhere (README?) about plan to shift this to rails, and link to work you started???

o.w. looks fine to me.

**Comment by tingulfsen** [comment]
Thanks for the comment. I updated the README.

**Comment by jkeck** [comment]
👍 from me, although I sort-of helped pair on this (if I remember correctly) so me may want other 👀.

**Pull request merged**